web/src/components/navbar/NavBarMobile.tsx

```diff
@@ -83,17 +83,19 @@ export const NavBarMobile = (props: Props): ReactElement => {
             <NavigatorLogo />
           )}
         </div>
-        <button
-          className="right-nav-menu-button radius-0"
-          data-testid="nav-menu-open"
-          aria-label="open menu"
-          onClick={(): void => {
-            analytics.event.mobile_hamburger_icon.click.open_mobile_menu();
-            open();
-          }}
-        >
-          <Icon className="font-sans-xl">menu</Icon>
-        </button>
+        {(!currentlyOnboarding() || !isAuthenticated) && (
```

Comment:
boulder - this logic is removing the hamburger menu from across the app.

Reply:
discussed: This is fine.